src/Async.js

```diff
@@ -85,6 +86,9 @@ const Async = React.createClass({
 				cache: initCache(nextProps.cache),
 			});
 		}
+		if (nextProps.value !== this.props.value) {
+			this.loadOptions('');
```

Comment:
Sure! So, if the database table being searched is large, the process of returning the entire result set could be very slow. A developer might therefore limit the query to, say, 100 records. If the record that your `value` property represents is the 101st record, it will not be returned by searching for `''`.